sources/packages/web/src/views/student/StudentOverawardsBalance.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <student-page-container layout-template="centered" hide-restriction="true">
+    <template #header>
+      <header-navigator
+        title="Home"
+        subTitle="Overawards Balance"
+        :routeLocation="{ name: StudentRoutesConst.STUDENT_DASHBOARD }"
+      />
+    </template>
+    <body-header-container :enableCardView="true">
```

Comment:
Do we need a `body-header-container` here since we do not have a header template defined?